### PR TITLE
Nav demo improvements

### PR DESCRIPTION
**Pull request by civerachb-cpr**
Expose the scan_topic in the amcl + gmapping demos. Unlike the other platforms, we don't have a RIDGEBACK_LASER_TOPIC env var to leverage, so stick with front/scan as the default.

### CODE REVIEW

**Review by tonybaltovski — APPROVED** [review]

File: ridgeback_navigation/launch/amcl_demo.launch [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <launch>
+  <arg name="scan_topic" default="front/scan" />
```
Comment (tonybaltovski) [inline ridgeback_navigation/launch/amcl_demo.launch:2]: Shouldn't this be `  <arg name="scan_topic" default="$(eval optenv('RIDGEBACK_LASER_TOPIC', 'front/scan'))" />`?

File: ridgeback_navigation/launch/gmapping_demo.launch [hunk 2]
```diff
@@ -1,7 +1,10 @@
 <launch>
+  <arg name="scan_topic" default="front/scan" />
```
Comment (tonybaltovski) [inline ridgeback_navigation/launch/gmapping_demo.launch:2]: Shouldn't this be `<arg name="scan_topic" default="$(eval optenv('RIDGEBACK_LASER_TOPIC', 'front/scan'))" />`?

File: ridgeback_navigation/launch/amcl_demo.launch (hunk 1 above)
Comment (civerachb-cpr) [reply ridgeback_navigation/launch/amcl_demo.launch:2]: From the description of the PR: "Unlike the other platforms, we don't have a RIDGEBACK_LASER_TOPIC env var to leverage, so stick with front/scan as the default."

File: ridgeback_navigation/launch/gmapping_demo.launch (hunk 2 above)
Comment (civerachb-cpr) [reply ridgeback_navigation/launch/gmapping_demo.launch:2]: See above

File: ridgeback_navigation/launch/amcl_demo.launch (hunk 1 above)
Comment (tonybaltovski) [reply ridgeback_navigation/launch/amcl_demo.launch:2]: Ops, thanks for the clarification.

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**